src/objective-c/change-comments.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+
+import re
+import sys
+
+
+if len(sys.argv) < 2:
+  print("Please provide at least one source file name as argument.")
+  quit()
+
+for file_name in sys.argv[1:]:
+
+  print("Modifying format of {file} comments in place...".format(
+      file = file_name,
+  ))
+
+
+  # Input
+
+  lines = []
```

Comment:
The value assigned to `lines` on this line cannot possibly be used before `lines` is reassigned to a different value. Isn't this dead code?